### PR TITLE
Add acceptFormat property to DigFileUploadView 

### PR DESCRIPTION
**Pull request by yij793**
changes: add one more property to DigFileUploadView to allow user choice the type of upload file type, such as image/png or application/vnd.ms-excel, if it is empty, will be no limit

### CODE REVIEW

**Comment by minsunny** [comment]
will update Dig.js too as this view is used Dig.js

File: src/foam/nanos/dig/DigFileUploadView.js [hunk 1]
```diff
@@ -103,6 +103,11 @@ foam.CLASS({
       name: 'dragActive',
       value: false
     },
+    {
+      class: 'String',
+      name: 'acceptFormat',
+      value: false
```
Comment (TW80000) [inline src/foam/nanos/dig/DigFileUploadView.js]: `false` doesn't make sense as a default value for a String property.

Comment (yij793) [reply src/foam/nanos/dig/DigFileUploadView.js]: I removed the value, just wondering should we make this value to 'application/vnd.ms-excel, text/csv'?

Comment (TW80000) [reply src/foam/nanos/dig/DigFileUploadView.js]: Yeah, otherwise anything that depended on the default value will break.

Comment (yij793) [reply src/foam/nanos/dig/DigFileUploadView.js]: Ok, changed, please review

**Review by TW80000 — APPROVED** [review]
I have no issue with this.

**Pull request merged**